src/Codeception/Extension/MultiDb.php

```diff
@@ -573,11 +573,11 @@ public function dontSeeInDb($table, $criteria)
      *
      * @return array an array of rows ( depending on the $fetchPdoArgs given )
      */
-    public function getFromDb($table, $criteria, $fields = null, $fetchPdoArgs = array(\PDO::FETCH_ASSOC))
+    public function getFromDb($table, $criteria, $limit = 1, $fields = null, $fetchPdoArgs = array(\PDO::FETCH_ASSOC))
```

Comment:
I would be concerned to change the order of arguments; i.e by adding a new argument in the middle.
It will imply that any client code that specifies `$fields` or `$fetchPdoArgs` in calls to `getFromDb()` will have to change too.

Please consider the form below instead.
```
public function getFromDb($table, $criteria, $fields = null, $fetchPdoArgs = array(\PDO::FETCH_ASSOC, $limit = 1))
```

To make life easier, it might be worth adding a new function, something like `getOneRecordFromDb()`; better ideas for the name, most welcome.

@abulford any thoughts?